packages/webview_flutter/tizen/src/webview_factory.cc

```diff
@@ -21,36 +21,38 @@
 #include "webview_flutter_tizen_plugin.h"
 
 WebViewFactory::WebViewFactory(flutter::PluginRegistrar* registrar,
-                               flutter::TextureRegistrar* textureRegistrar)
-    : PlatformViewFactory(registrar), texture_registrar_(textureRegistrar) {
+                               flutter::TextureRegistrar* texture_registrar)
+    : PlatformViewFactory(registrar), texture_registrar_(texture_registrar) {
   char* path = app_get_data_path();
+  std::string path_string;
   if (!path || strlen(path) == 0) {
-    path = "/tmp/";
```

Comment:
@seungsoo47 here is the line which was causing a releasing a non-heap memory with free() - refer to line 39.

I've removed this condition and replaced conditional memory releasing with a std::string-based stack memory management.

Reply:
👍